frontend/src/pages/ResetPassword.vue

```diff
@@ -88,7 +88,11 @@ export default {
             ? this.$t('message.checkEmail')
             : this.$t('message.reset')
           this.messageButtonText = this.$t('login')
-          this.messageButtonLinktTo = '/login'
+          if (this.$route.params.code) {
+            this.messageButtonLinktTo = `/login/${this.$route.params.code}`
+          } else {
+            this.messageButtonLinktTo = '/login'
+          }
```

Comment:
Why ever, with me that does not work on Safari for a new registration with a transaction link.

Reply:
We have to fix this in the future.

Reply:
I created an issue